lib/src/video_player/method_channel_video_player.dart

```diff
@@ -446,6 +446,7 @@ class MethodChannelVideoPlayer extends VideoPlayerPlatform {
           return VideoEvent(
             eventType: VideoEventType.exitingPIP,
             key: key,
+            wasPlaying: map['wasPlaying'] as bool,
```

Comment:
```suggestion
            wasPlaying: map['wasPlaying'] as bool?,
```
It can make an exception on Android. Please change the type cast for safe.